lib/qx/tool/compiler/ClassFile.js

```diff
@@ -286,7 +286,7 @@ qx.Class.define("qx.tool.compiler.ClassFile", {
               filename: t.getSourcePath(),
               sourceMaps: true,
               "presets": [
-                [ require.resolve("@qooxdoo/preset-env"), { "modules": false } ]
```

Comment:
i realise this PR is closed, but were you able to check that qooxdoo/preset-env is no longer required?  There was a bug which had a big performance impact, details at https://github.com/qooxdoo/qooxdoo-preset-env/blob/master/README.md

Reply:
They have reworked this in babel 7:

 if (block.body) {
      const declarPaths = this.blockPath.get("body");
      for (let i = 0; i < block.body.length; i++) {
        addDeclarationsFromChild(declarPaths[i]);
      }
    }